controller/repliesController.js

```diff
@@ -159,104 +162,74 @@ const getReplyVotes = async (req, res, next) => {
 
     const reply = await Replies.findById(replyId);
 
-    // Check to see if the reply exists.
+// Check to see if the reply exists.
     if (!reply) {
       return next(
         new CustomError(
           404,
-          `Reply with the ID ${replyId} does not exist or has been deleted`
+          'Reply with the ID ${replyId} does not exist or has been deleted'
```

Comment:
Change this back to bacticks (`) instead of single quotes(') for interpolation to work.

Reply:
Done